contracts/root/ExitManager.sol

```diff
@@ -71,24 +71,6 @@ contract ExitManager is RootChainable {
   // Public functions
   //
 
-  /**
-  * @dev Returns information about an exit.
-  * @param _utxoPos Position of the UTXO in the chain.
-  * @return A tuple representing the active exit for the given UTXO.
-  */
-  function getExit(uint256 _utxoPos)
```

Comment:
This function is redundant since exits mapping is public. See corresponding client contract changes above. Less code is gas savings in deployment.

Reply:
We have added this function so that users/clients don't have to look up to the contracts to know what is being returned as we cant return the whole struct. [See More](https://github.com/ethereum/web3.js/issues/1728)

Reply:
How is lookup a problem?
`ExitManager.methods.exits(1).call();` will return JSON data of the form
```
{ '0': '0x4aA2032526286188d7ef812c2d4702E0AeffF0ad',
  '1': '0x0000000000000000000000000000000000000001',
  '2': '100',
  '3': true,
  owner: '0x4aA2032526286188d7ef812c2d4702E0AeffF0ad',
  token: '0x0000000000000000000000000000000000000001',
  amountOrTokenId: '100',
  burnt: true }
```